src/FrequencyMapAgent.cpp

```diff
@@ -364,22 +364,33 @@ namespace geopm
     std::vector<std::pair<std::string, std::string> > FrequencyMapAgent::report_host(void) const
     {
         std::vector<std::pair<std::string, std::string> > result;
-        std::ostringstream oss;
-        oss << std::setprecision(M_PRECISION) << std::scientific;
-        for (const auto &region : m_hash_freq_map) {
-            oss << "\n    0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
-            oss << region.first;
-            oss << std::setfill('\0') << std::setw(0) << std::scientific;
-            oss << ": " << region.second;
-        }
-        for (const auto &region : m_default_freq_hash) {
-            oss << "\n    0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
-            oss << region;
-            oss << std::setfill('\0') << std::setw(0) << std::scientific;
-            oss << ": " << m_default_freq;
+
+        std::map<uint64_t, double> full_map(m_hash_freq_map);
+        std::for_each(m_default_freq_hash.begin(), m_default_freq_hash.end(), [&](uint64_t key)
+        {
+            full_map.insert({ key, m_default_freq });
+        });
+
+        std::map<std::string, Json> temp_map;
+        for (const auto &region : full_map)
+        {
+            // Json object(int(region.first));
+            // std::string key_string = object.dump();
+            // key_string.erase(std::remove(key_string.begin(), key_string.end(), '"'), key_string.end());
+            // temp_map[key_string] = region.second;
+            std::ostringstream key_ss;
+            key_ss << "0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
```

Comment:
```suggestion
            key_ss << "\n    0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
```

This needs to remain so that the indentation in the report is still correct.